www/dialonce.js

```diff
@@ -0,0 +1,9 @@
+var exec = require('cordova/exec');
+
+var dialonce = {
+    requestPermissions:function() {
```

Comment:
Please add some jsdoc here, also the key and value should be formatted properly

e.g `requestPermissions: function() {`